techempower/manifests/quarkus-resteasy-hibernate.yaml

```diff
@@ -50,7 +50,7 @@ spec:
   type: NodePort
   ports:
     - port: 8080
-      nodePort: 32334
+      nodePort: 32434
```

Comment:
Why does this need to be a fixed port number? We can get the port number as we do in the demo [script](https://github.com/kruize/autotune-demo/blob/main/minikube_demo_setup.sh#L260).

Reply:
Removed the fixed number. openshift runs doesn't require port. 
Will update the required port changes in minikube PR #43.